topo_processor/util/tests/transfer_collection_test.py

```diff
@@ -25,19 +26,40 @@ def test_fail_on_duplicate_assets(setup):
     target = setup
     collection = Collection("fake_title")
     collection.description = "fake_description"
-    collection.license = "face_license"
+    collection.license = "fake_license"
     item = Item("item_id")
     item.datetime = datetime.now()
     collection.add_item(item)
     item.collection = collection
 
     cog_1 = Asset("./test_data/tiffs/SURVEY_1/CONTROL.tiff")
     cog_1.target = "fake_title/fake_target.tiff"
+    cog_1.key_name = AssetKey.Visual.value
```

Comment:
Instead of using `enum.value` you can define AssetKey(str, Enum), so the `enum` will return the `string` value. i.e. `AssetKey.Visual == "visual" # True`